internal/controller/feature/features.go

```diff
@@ -58,6 +58,12 @@ const (
 	// This feature gate must be used together with LiteralCertificateSubject webhook feature gate.
 	// See https://github.com/cert-manager/cert-manager/issues/3203 and https://github.com/cert-manager/cert-manager/issues/4424 for context.
 	LiteralCertificateSubject featuregate.Feature = "LiteralCertificateSubject"
+
+	// Alpha: v1.9.1
```

Comment:
I think this could be 1.10 or a 1.11 alpha release if it does not make it into 1.10
I don't think we would add a new feature in a patch release

Reply:
Changed it to 1.10 for now.